src/background/stores/dev-tools-store.ts

```diff
@@ -19,6 +19,7 @@ export class DevToolStore extends BaseStoreImpl<DevToolState> {
     public getDefaultState(): DevToolState {
         const defaultValues: DevToolState = {
             isOpen: false,
+            inspectElementRequestId: 0,
```

Comment:
can we just send a message back from devtools once the inspect is successful? if inspect is successfule, background can clear the inspectedElement value rite?

Reply:
That's more complex to do... I now this seems hacky, but it's way simpler